README.md

```diff
@@ -16,42 +16,39 @@ If you do not have such hardware available, there is also the option to use a Ze
 
 In the following, we give a more detailed overview of the different involved components.
 
-### Horn Service Implementation
+### Horn Service Kuksa
```

Comment:
Good point to rename this service to Kuksa since there can obviously be different version with other backends.